### PR TITLE
Drop _ from non-private functions

### PR DESCRIPTION
**Pull request by deepesh-kn**
Renamed all the files starting with `_` in contract interacts.
Fixes: #79

### CODE REVIEW

**Review by schemar — APPROVED** [review]
LGTM 🙌

File: src/ContractInteract/EIP20CoGateway.js [hunk 1]
```diff
@@ -67,7 +67,8 @@ class EIP20CoGateway {
     this.getUtilityToken = this.getUtilityToken.bind(this);
     this.isRedeemAmountApproved = this.isRedeemAmountApproved.bind(this);
     this.redeem = this.redeem.bind(this);
-    this._redeemRawTx = this._redeemRawTx.bind(this);
+    this.redeemRawTx = this.redeemRawTx.bind(this);
+    this.progressRedeemRawTx = this.progressRedeemRawTx.bind(this);
```
Comment (0xsarvesh) [inline src/ContractInteract/EIP20CoGateway.js:71]:
```suggestion
    this.progressRedeemRawTx = this.progressRedeemRawTx.bind(this);
    this.approveRedeemAmount = this.approveRedeemAmount.bind(this);
    this.progressRedeem = this.progressRedeem.bind(this);
```

File: src/ContractInteract/EIP20Gateway.js [hunk 2]
```diff
@@ -68,6 +68,8 @@ class EIP20Gateway {
     this.getValueTokenContract = this.getValueTokenContract.bind(this);
     this.isBountyAmountApproved = this.isBountyAmountApproved.bind(this);
     this.isStakeAmountApproved = this.isStakeAmountApproved.bind(this);
+    this.confirmRedeemIntentRawTx = this.confirmRedeemIntentRawTx.bind(this);
+    this.progressUnstakeRawTx = this.progressUnstakeRawTx.bind(this);
```
Comment (0xsarvesh) [inline src/ContractInteract/EIP20Gateway.js:72]:
```suggestion
    this.progressUnstakeRawTx = this.progressUnstakeRawTx.bind(this);
    this.confirmRedeemIntent = this.confirmRedeemIntent.bind(this);
    this.progressUnstake = this.progressUnstake.bind(this);
```

File: src/ContractInteract/EIP20Token.js [hunk 3]
```diff
@@ -44,7 +44,7 @@ class EIP20Token {
     }
 
     this.approve = this.approve.bind(this);
-    this._approveRawTx = this._approveRawTx.bind(this);
+    this.approveRawTx = this.approveRawTx.bind(this);
```
Comment (0xsarvesh) [inline src/ContractInteract/EIP20Token.js:47]:
```suggestion
    this.approveRawTx = this.approveRawTx.bind(this);
    this.balanceOf = this.balanceOf.bind(this);
```

File: src/ContractInteract/OSTPrime.js [hunk 4]
```diff
@@ -36,13 +36,13 @@ class OSTPrime {
...
-    this._approveRawTx = this._approveRawTx.bind(this);
+    this.approveRawTx = this.approveRawTx.bind(this);
     this.allowance = this.allowance.bind(this);
     this.isAmountApproved = this.isAmountApproved.bind(this);
     this.wrap = this.wrap.bind(this);
-    this._wrapRawTx = this._wrapRawTx.bind(this);
+    this.wrapRawTx = this.wrapRawTx.bind(this);
     this.unwrap = this.unwrap.bind(this);
-    this._unwrapRawTx = this._unwrapRawTx.bind(this);
+    this.unwrapRawTx = this.unwrapRawTx.bind(this);
```
Comment (0xsarvesh) [inline src/ContractInteract/OSTPrime.js:45]:
```suggestion
    this.unwrapRawTx = this.unwrapRawTx.bind(this);
    this.balanceOf = this.balanceOf.bind(this);
```

**Review by 0xsarvesh — APPROVED** [review]
LGTM 🐇 🐎 

Minor feedback.
Method binding to `this` is missing for some classes. Please check inline comment.

**Pull request merged**